src/commands/tileindex-validate/tileindex.validate.ts

```diff
@@ -522,3 +560,38 @@ export async function validate8BitsTiff(tiff: Tiff): Promise<void> {
     throw new Error(`${tiff.source.url.href} is not a 8 bits TIFF`);
   }
 }
+
+export function* iterateMapSheets(bounds: BBox, gridSize: GridSize): Generator<string> {
+  const minX = Math.min(bounds[0], bounds[2]);
+  const maxX = Math.max(bounds[0], bounds[2]);
+  const minY = Math.min(bounds[1], bounds[3]);
+  const maxY = Math.max(bounds[1], bounds[3]);
+
+  const tilesPerMapSheet = Math.floor(MapSheet.gridSizeMax / gridSize);
+
+  const tileWidth = Math.floor(MapSheet.width / tilesPerMapSheet);
+  const tileHeight = Math.floor(MapSheet.height / tilesPerMapSheet);
+
+  let tile: string = '';
+  const stepsX = calculateStep(minX, maxX, MapSheet.origin.x, tileWidth);
+  const stepsY = calculateStep(minY, maxY, MapSheet.origin.y, tileHeight);
+
+  for (let stepY = 0; stepY < stepsY; stepY += 1) {
+    const y = maxY - stepY * tileHeight;
+    for (let stepX = 0; stepX < stepsX; stepX += 1) {
+      const x = minX + stepX * tileWidth;
+      try {
+        tile = getTileName(x, y, gridSize);
+      } catch (e) {
+        if (e instanceof RangeError) {
```

Comment:
`instanceof` and Errors is somewhat risky pattern in javascript, in this particualr use case it would work, but its not a pattern we should be using.

I think we should just get rid of the is known check, we could turn it into a log and see how often its hit.